### PR TITLE
Make signed out navigation for Non-English users the same as current English users

### PR DESCRIPTION
**Pull request by juanmanzojr**
Makes the navigation for signed out Non-English users the same as the current English signed out users.
### English signed out user
Screenshot 2023-09-28 at 3 50 09 PM

### Non-English signed out user
Screenshot 2023-09-28 at 3 50 21 PM

## Links

- jira ticket: [here](https://codedotorg.atlassian.net/jira/software/c/projects/ACQ/boards/64?modal=detail&selectedIssue=ACQ-937)

## Testing story
Local

## Follow-up work
I will work on making signed in teacher navigation the same for english and non english users.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: lib/cdo/hamburger.rb [hunk 1]
```diff
@@ -297,8 +297,12 @@ def self.get_header_contents(options)
     ]
 
     non_en_signed_out_links = [
```
Comment (dmcavoy) [inline lib/cdo/hamburger.rb]: Wonder: Can we instead remove the special casing of english vs non-english so these stay consistent?

Comment (juanmanzojr) [reply lib/cdo/hamburger.rb:299]: Yes! Ill make sure to do this for the others as well

Comment (juanmanzojr) [reply lib/cdo/hamburger.rb]: Yes! Ill make sure to do this from here on out

File: lib/cdo/hamburger.rb [hunk 2]
```diff
@@ -320,10 +318,11 @@ def self.get_header_contents(options)
         header_links.concat(about_intl)
       end
     else
+      header_links = signed_out_links
       if options[:language] == "en"
-        header_links = en_signed_out_links
+        header_links.concat(about)
       else
-        header_links.concat(non_en_signed_out_links).concat(about_intl)
+        header_links.concat(about_intl)
```
Comment (dmcavoy) [inline lib/cdo/hamburger.rb]: We no longer need a different international about. Everyone should get the normal about

**Review by TurnerRiley — APPROVED** [review]

**Pull request merged**